OpenRA.Mods.Common/Traits/ExternalCapturable.cs

```diff
@@ -24,6 +24,9 @@ public class ExternalCapturableInfo : ITraitInfo
 		[Desc("Seconds it takes to change the owner.", "You might want to add a ExternalCapturableBar: trait, too.")]
 		public readonly int CaptureCompleteTime = 15;
 
+		[Desc("Whether autotargeting ignores this actor while it is being captured by an ally.")]
+		public readonly bool AutoTargetIgnores = false;
```

Comment:
@Mailaender 
> I think you can safely set this to true. At least for the Monster Tank Madness mission.

I am not sure what that means in relation to the PR.
I thought that this was the wrong place to implement a setting for a specific mission. Which led me to believe that your comment was just meant as a note to users of the field. Is that correct?
Or do you think changes should be made to the PR?